.editorconfig

```diff
@@ -4,9 +4,11 @@
 
 root = true
 
+[*]
```

Comment:
* Should we enforce lf line endings for all documents?
* Should we try and remove duplicate entries by adding to [*]? E.g. insert_final_newline/trim_trailing_whitespace/charset

Reply:
@dalemorson thoughts?